app/src/main/java/org/wikipedia/feed/announcement/AnnouncementCardView.java

```diff
@@ -104,6 +104,7 @@ public AnnouncementCardView(@NonNull Context context) {
             LinearLayout.LayoutParams layoutParams = (LinearLayout.LayoutParams) container.getLayoutParams();
             layoutParams.setMarginStart(0);
             layoutParams.setMarginEnd(0);
```

Comment:
Not needed anymore

Reply:
Remove the lines will not work, because `setMargins()` only applies for `left`, `top`, `bottom`, `right`.

Reply:
Couldn't we avoid all of this by just saying  `container.setLayoutParams(new  LinearLayout.LayoutParams(MATCH_PARENT, MATCH_PARENT))`

Reply:
You are correct. Done!